patches/0073-libavfi-dnn-add-LibTorch-as-one-of-DNN-backend.patch

```diff
@@ -388,6 +396,12 @@ index 0000000000..4f1395414c
 +    return NULL;
 +}
 +
++static void deleter(void *arg)
++{
++    if (arg)
++        av_free(arg);
```

Comment:
Needn't check whether arg is NULL, see https://github.com/FFmpeg/FFmpeg/blob/master/libavutil/mem.h#L350

Reply:
Is it possible to use av_freep() instead ?

Reply:
I update the PR. Thanks for your advice.